x/oracle/keeper/historic_price.go

```diff
@@ -134,63 +145,63 @@ func (k Keeper) SetHistoricMedianDeviation(
 }
 
 // MedianOfHistoricMedians calculates and returns the median of the last stampNum
-// historic medians.
+// historic medians as well as the amount of medians used to calculate that median.
 func (k Keeper) MedianOfHistoricMedians(
 	ctx sdk.Context,
 	denom string,
 	numStamps uint64,
-) (sdk.Dec, error) {
+) (sdk.Dec, uint32, error) {
 	medians := k.HistoricMedians(ctx, denom, numStamps)
```

Comment:
```suggestion
	medians := k.HistoricMedians(ctx, denom, numStamps)
	if len(medians) == 0 {
		sdk.ZeroDec(), 0, nil
	}
```
Prevents error return "empty list passed in" when dealing with this case.

Can also do this with the other functions, and add to each of their godoc comments:
> Also returns the actual number of median stamps retrieved, which may be less than
> numStamps requested. If no medians are available, all returns are zero and error is nil.

Reply:
Done👍